pkg/settings/setting.go

```diff
@@ -154,9 +154,12 @@ func GetEnvKey(key string) string {
 func getMetadataConfig() string {
 	key := GetEnvKey("server-version")
 	rancherVersion := os.Getenv(key)
-	branch := "dev"
-	if releaseServerVersion(rancherVersion) {
-		branch = "master"
+	branch := os.Getenv("RANCHER_METADATA_BRANCH")
```

Comment:
@cjellick are there any issues with trying to use a default value of an env var as empty?

Reply:
spoke with @cjellick offline and there is no downside to this concept